app/services/config-persistence/config.ts

```diff
@@ -236,12 +236,17 @@ export class ConfigPersistenceService extends PersistentStatefulService<IScenesC
 
 
   switchToBlankConfig(configName = DEFAULT_SCENES_COLLECTION_NAME) {
+    this.switchToBlankConfig(configName);
```

Comment:
Infinity recursion here. Looks like in some cases app will not start

Reply:
Yikes, meant to call the new method I created.  Good catch!